src/components/Fields/TextArea.js

```diff
@@ -51,6 +52,7 @@ const defaultProps = {
     isValid: null,
     label: null,
     name: null,
+    autoComplete: null,
```

Comment:
Wouldn't be good to have a default value assigned to it?

Reply:
If we don't pass a default value it will just use browser's default, which is to have it on. So I don't see the need for us to add it.

Reply:
No, it's better to have no value which will make React not output the `autocomplete` attribute at all.
This will be valid HTML and consistent with the HTML spec - not having the `autocomplete` attribute means that the browser will look at the parent `<form>` tag to decide whether to use autocomplete or not.